patch-place-break/patch-place-break-config/src/test/java/fr/djaytan/minecraft/jobsreborn/patchplacebreak/config/annotated/CredentialsValidatingPropertiesTest.java

```diff
@@ -25,9 +25,10 @@
 package fr.djaytan.minecraft.jobsreborn.patchplacebreak.config.annotated;
```

Comment:
A merge between CredentialsValidatingPropertiesTest and CredentialsValidatingPropertiesIT